### ISSUE TITLE
Accessibility: Add the aria-current attribute to the current language in the selection widget

### ISSUE THREAD

**Issue by ivnc** [polylang/polylang#1621]
**Describe the bug**

In the language switching widget, the current language is transmitted visually, but not to assistive technologies such as screen readers. This deprives disabled and other users with different difficulties that rely on assistive technologies of this information.

**To Reproduce**

1. Place a language switching widget in some part of the website.
2. Examine the code, the currently active language is identified with a CSS class, but not with aria-current.

**Expected behavior**
The list item or link corresponding to the currently active language should be marked with the attribute aria-current="true". This could help in complying with the success criterion 4.1.2 from the WCAG.

**Environment (please complete the following information):**
 - Device: [e.g. Desktop, iPhone...]
 - OS: [e.g. Unbuntu, iOS...]
 - Browser [e.g. Chrome, Safari...]

**Additional context, screenshots**
